services/solr-names-updater/src/solr_names_updater/config.py

```diff
@@ -0,0 +1,138 @@
+# Copyright © 2019 Province of British Columbia
```

Comment:
why is config.py in here twice?

Reply:
the one in the root was required due to how namex api was structured.  `names-pay` has two config files as well.  I'm assuming it does this to work around this issue.  I've moved the `config.py` file in the namex api and made some updates so that we can use one `config.py` in this queue listener.